plugin/evm/evm_call_test.go

```diff
@@ -0,0 +1,91 @@
+package evm
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/flare-foundation/coreth/accounts/abi"
+	"github.com/flare-foundation/coreth/accounts/abi/bind"
+	"github.com/flare-foundation/coreth/accounts/abi/bind/backends"
+	"github.com/flare-foundation/coreth/core"
+)
+
+func TestEVMCall_Execute(t *testing.T) {
+
+	key, err := crypto.GenerateKey()
+	require.NoError(t, err)
+
+	auth, err := bind.NewKeyedTransactorWithChainID(key, big.NewInt(1337))
+	require.NoError(t, err)
+
+	balance := new(big.Int)
+	balance.SetString("100000000000000000000", 10)
```

Comment:
You can also use `SetUint64` and use a number literal or actual `uint64` variable.